app/assets/javascripts/solidus_paypal_braintree/frontend.js

```diff
@@ -0,0 +1,11 @@
+// This is a manifest file that'll be compiled into including all the files listed below.
+// Add new JavaScript/Coffee code in separate files in this directory and they'll automatically
+// be included in the compiled file accessible from http://example.com/assets/application.js
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// the compiled file.
+//
+//= require ./constants
+//= require ./promise
+//= require ./client
+//= require ./hosted_form
+//= require ./paypal_button
```

Comment:
Using relative paths makes it impossible to override the individual files (what we need to do in our app, because we load JS differently).

Changing this to

```js
//= require solidus_paypal_braintree/constants
//= require solidus_paypal_braintree/promise
//= require solidus_paypal_braintree/client
//= require solidus_paypal_braintree/hosted_form
//= require solidus_paypal_braintree/paypal_button
```

makes overwriting possible.